.prospector.yaml

```diff
@@ -0,0 +1,17 @@
+test-warnings: true
+doc-warnings: false
+member-warnings: false
+
+ignore-paths:
+  - docs
+ignore-patterns:
+  - (^|/)skip(this)?(/|$)
+autodetect: true
+#max-line-length: 88
+
+pylint:
+  disable:
+    # parameter_validations library methods give this error
+    - no-value-for-parameter
```

Comment:
can we ignore this only in the places where we need to do so ?

Reply:
Yes I removed the general rule for no-value-for-parameter and added it inline wherever it is needed with comments